rpc/cosmos/net_info.go

```diff
@@ -80,8 +80,9 @@ type netInfo struct {
 	} `json:"result"`
 }
 
-func GetNpeers() (string, error) {
-	req, err := http.NewRequest(http.MethodGet, "http://localhost:26657/net_info", nil)
+func GetNpeers(tenderPort uint) (string, error) {
+	url := fmt.Sprintf("http://localhost:%d/net_info", tenderPort)
```

Comment:
I think we need to consider to get the data from another rpc endpoint, not only localhost. 
please refer to the another plugins.

- https://github.com/dsrvlabs/vatz-plugin-cosmoshub/blob/main/rpc/cosmos/api.go#L55
- https://github.com/dsrvlabs/vatz-plugin-cosmoshub/blob/main/rpc/cosmos/health.go#L12

Reply:
governance alarm plugin can also get the data from localhost only.

- https://github.com/dsrvlabs/vatz-plugin-cosmoshub/blob/main/rpc/cosmos/governance.go#L58

Reply:
@heejin-github , I added an option for rpc address and removed tendermint port.
Thanks!